src/components/Container/index.tsx

```diff
@@ -61,17 +68,12 @@ const Container: FunctionComponent<ContainerProps> = ({
 }) => (
   <StyledContainer {...props}>
     <FlexBox justifyContent="space-between">
-      <Box
-        display="flex"
-        alignItems="center"
-        justifyContent="flex-start"
-        mb={1}
-      >
+      <StyledTitleContainer>
```

Comment:
Why not using flexbox ?

Reply:
Used Flexbox but need to keep StyledTitleContainer for the margin-bottom